src/Test/L0/Worker/Build/TrackingManagerL0.cs

```diff
@@ -520,5 +520,12 @@ public void UpdatesTrackingConfigJobRunProperties()
                 Assert.True(DateTimeOffset.Now.AddSeconds(1) >= config.LastRunOn);
             }
         }
+
+        //removes CR so the test can run on Linux, even if it was synced on a Windows drive share
+        //this is used when Linux VM is building from a folder mounted on a Windows box
+        private string TrimCR(string value)
```

Comment:
will rename to "RemoveCR". we can discuss a better solution if the issue comes up again.